mapie/estimators.py

```diff
@@ -226,25 +226,22 @@ def _check_alpha(self, alpha: Union[float, Iterable[float]]) -> np.ndarray:
         Raises
         ------
         ValueError
-            [description]
+            If alpha is not a float or an Iterable of floats between 0 and 1.
         """
-        flag = 0
-        if isinstance(alpha, Iterable):
-            for aa in alpha:
-                if not isinstance(aa, float) or not 0 < aa < 1:
-                    flag += 1
-        elif isinstance(alpha, float):
-            if not 0 < alpha < 1:
-                flag += 1
+        if isinstance(alpha, float):
+            alpha = np.array([alpha]).copy()
+        elif isinstance(alpha, Iterable):
```

Comment:
Question bête : on ne peut pas simplement faire un check `isinstance(alpha, Iterable[float])` ?

Reply:
J'avais essayé mais ça n'a pas l'air possible: "TypeError: Subscripted generics cannot be used with class and instance checks"